kouta-backend/src/main/scala/fi/oph/kouta/client/kayttooikeusClient.scala

```diff
@@ -30,7 +30,7 @@ trait KayttooikeusClient extends HttpClient with CallerId with Logging {
   private lazy val casConfig: CasConfig = new CasConfig.CasConfigBuilder(
     config.username,
     config.password,
-    urlProperties.url("cas.url"),
+    config.casUrl,
```

Comment:
Oppijanumerorekisteri- ja kayttooikeusClientien CAS-autentikaatio _lokaalissa ympäristössä_ on todennäköisesti rikkoutunut java-cassin käyttöönoton yhteydessä: https://github.com/Opetushallitus/kouta-backend/pull/399. Jostain syystä `urlProperties.url("cas.url")` ei toimi lokaalisti, koska templaten `{{cas_url}}` päätyy kirjaimellisesti tuossa muodossa URLiin: "{{cas_url}}/v1/tickets could not be parsed into a proper Uri, missing scheme". En saanut korjattua tätä konffitiedostoja muuttamalla.